### PR TITLE
OSOE-93: Add a UI test to Lombiq.Privacy

### PR DESCRIPTION
**Pull request by dministro**
OSOE-93

### CODE REVIEW

**Comment by sarahelsaig** [comment]
You have a merge conflict

File: test/Lombiq.OSOCE.Tests.UI/Tests/BehaviorPrivacyTests.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            // Then should work with razor based theme
+            ExecuteTestAfterSetupAsync(
+                async context =>
+                {
+                    await context.SelectThemeAsync("TheTheme");
+                    await context.SignInDirectlyAsync();
+                    await context.TestConsentBannerAsync();
+                },
+                browser)
+            );
```
Comment (sarahelsaig) [inline test/Lombiq.OSOCE.Tests.UI/Tests/BehaviorPrivacyTests.cs]: These two are running in parallel. Is that safe? Also the code is identical besides the theme name, so please extract into a method.

**Review by sarahelsaig — CHANGES_REQUESTED** [review]

**Review by sarahelsaig — CHANGES_REQUESTED** [review]
Add Lombiq.ChartJs.Samples and Lombiq.Privacy.Samples to the [NuGet csproj](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/issue/OSOE-93/NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj) and invoke the tests for both project from [BehaviorModuleTests.cs](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/issue/OSOE-93/NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/Tests/BehaviorModuleTests.cs) as well. 
(But first push a new tag for Lombiq.Privacy.)

**Comment by dministro** [comment]
> Add Lombiq.ChartJs.Samples and Lombiq.Privacy.Samples to the [NuGet csproj](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/issue/OSOE-93/NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj) and invoke the tests for both project from [BehaviorModuleTests.cs](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/issue/OSOE-93/NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/Tests/BehaviorModuleTests.cs) as well. (But first push a new tag for Lombiq.Privacy.)

I did this [here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/105/commits/075218320ae89e3fe31c5a4e627a8c7bd5df83e0) but CI `Build and Test (NuGet solution) (Developer)` failed because of the prerelease package versions.

What is the correct process to make this?

**Comment by sarahelsaig** [comment]
I have checked out the commit you linked, run the command in the failed CI's log and it worked fine for me with no errors. In principle the errors make no sense, because you have the correct versions in your project files. Could be that you committed too soon and the NuGet cache wasn't updated yet, but normally that results in a "version not found" type of message so I'm just speculating.

Give it another try, but before you push to origin make sure it works for you locally by running
```powershell
dotnet build Lombiq.OSOCE.NuGet.sln --configuration Release --version-suffix 615 /p:Version="1.0.615.0" /p:RunAnalyzersDuringBuild=true /p:TreatWarningsAsErrors=true -warnaserror /nodeReuse:false /p:Retries=4 /p:RetryDelayMilliseconds=1000 /p:NoWarn=AD0001%3BNETSDK1179 -warnAsMessage:MSB3026
```
inside the solution's NuGetTest directory. If it works for you locally and still fails on the server, then open a thread in Teams in the infrastructure channel.

File: NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/.htmlvalidate.json [hunk 2]
```diff
@@ -14,5 +14,16 @@
...
+  "elements": [
+    "html5",
+    {
+      "form": {
+        "attributes": {
+          "method": [ "get", "post", "GET", "POST" ]
+        }
+      }
+    }
+  ]
```
Comment (sarahelsaig) [inline NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/.htmlvalidate.json]: Do you still need these? The `dev` of Lombiq.Tests.UI [already uses html-validation v.7.1.1](https://github.com/Lombiq/UI-Testing-Toolbox/blob/beebd4b322cf520c060fe409955b33ac6681a7c8/Lombiq.Tests.UI/Services/HtmlValidationConfiguration.cs#L36).

**Review by sarahelsaig — CHANGES_REQUESTED** [review]

**Pull request merged**